runtime/thunk/thunk.ts

```diff
@@ -43,25 +45,29 @@ export async function findAndExecutePackFunction<T extends FormulaSpecification>
   manifest: BasicPackDefinition,
   executionContext: ExecutionContext | SyncExecutionContext,
   shouldWrapError: boolean = true,
+  updates?: Array<SyncUpdate<any, any, any>>,
```

Comment:
Not married to this approach, but it seemed safest from a backwards/forwards compatibility point of view.

Reply:
I think updating the thunk file is probably safe. the runtime files are
- pack bundle, uploaded by the maker.
- the lambda bootstrap bundle and the thunk lambda which are both injected by the upload api. these two are the ones you are changing. they should always match each other.

Reply:
The interface of `executeThunk` needs to remain backwards-compatible, but `findAndExecutePackFunction` is only called internally, so it seems like that interface can change as we want? Seems like we maybe want start cleaning it up and pass a param dict.